README.md

```diff
@@ -16,52 +16,53 @@ To add the driver to your project, add the following Maven dependency:
 </dependency>
 ```
 
-The class name for this driver is:
+Replace `${jdbc.version}` with the actual version of the driver. It's good practice to manage this version in a properties section of your pom.xml.
 
-```
-com.salesforce.datacloud.jdbc.DataCloudJDBCDriver
-```
+### Driver Class
```

Comment:
I don't think we need this section. The "Driver Class (Reiterated)" section below should be sufficient